test_schemas/en/test_textfield_suggestions.json

```diff
@@ -34,17 +34,40 @@
                                 "answers": [
                                     {
                                         "id": "country-answer",
-                                        "label": "What is your country of birth?",
+                                        "label": "Country",
                                         "mandatory": false,
-                                        "suggestions_url": "https://cdn.eq.census-gcp.onsdigital.uk/data/v5.0.0/gb/en/countries-of-birth.json",
+                                        "suggestions": {
+                                            "url": "https://cdn.eq.census-gcp.onsdigital.uk/eq-lookup-suggestions-data/v5.3.0/gb/en/countries-of-birth.json",
+                                            "allow_multiple": false
+                                        },
                                         "type": "TextField"
                                     }
                                 ],
                                 "id": "country-of-birth-question",
-                                "title": "Title",
+                                "title": "What is your country of birth?",
                                 "type": "General"
-                            },
-                            "routing_rules": []
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "country-block-multiple-selections",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "country-multiple-selections-answer",
+                                        "label": "Country",
+                                        "mandatory": false,
+                                        "suggestions": {
+                                            "url": "https://cdn.eq.census-gcp.onsdigital.uk/eq-lookup-suggestions-data/v5.3.0/gb/en/countries-of-birth.json",
+                                            "allow_multiple": true
+                                        },
+                                        "type": "TextField"
+                                    }
+                                ],
+                                "id": "country-of-birth-multiple-selections-question",
+                                "title": "What is your country of birth?",
```

Comment:
I've just noticed that the question doesn't match what we're asking for - multiple countries. Also, the multiple selections doesn't work properly with countries of birth as some of the values already contain commas. We should use either the national identities list or passport countries.

Maybe the question could be "What passports do you hold?" (the answer label should then be "Passports") and use the passport countries list.

Reply:
It uses passport countries list now, new ids, label and title were added, thanks!